### PR TITLE
Clean up tie/forfeit timer messages + don't end GOTV on readyup-timeout

### PR DESCRIPTION
**Pull request by nickdnk**
This adjusts the messages printed in chat to use the `MM:SS` format instead of having multiple translations for seconds/minutes.

@ Translators. Required translations are:

When waiting for one team to ready:
`[TeamName] must be ready within [2:30] or they will forfeit the match.`

When waiting for both teams to be ready:
`Teams must be ready within [2:30] or the match will end in a tie.`

Just comment on the PR with the sentences and which language it is for. `[whatever]` indicates a variable.

Note that in this case, just to clarify, it is not the time of day, of course, but how many minutes and seconds remain.

### CODE REVIEW

File: translations/get5.phrases.txt [hunk 1]
```diff
@@ -68,20 +68,15 @@
...
     {
-        "#format"       "{1:s},{2:d}"
-        "en"            "{1} has {2} minutes left to ready up or they will forfeit the match."
-    }
-    "SecondsToForfeitInfoMessage"
-    {
-        "#format"       "{1:s},{2:d}"
-        "en"            "{1} has {2} seconds left to ready up or they will forfeit the match."
+        "#format"       "{1:s},{2:s}"
+        "en"            "{1} must be ready within {2} or they will forfeit the match."
```
Comment (SidiBecker) [inline translations/get5.phrases.txt:74]: pt:  "{1} deve estar pronta dentro de {2} ou perderá a partida."

File: translations/get5.phrases.txt [hunk 2]
```diff
@@ -68,20 +68,15 @@
...
     }
-    "10SecondsToForfeitInfoMessage"
+    "TeamsMustBeReadyOrTie"
     {
         "#format"       "{1:s}"
-        "en"            "{1} has 10 seconds to ready up or forfeit the match."
+        "en"            "Teams must be ready within {1} or the match will end in a tie."
```
Comment (SidiBecker) [inline translations/get5.phrases.txt:79]: pt:  "As equipes devem estar prontas em {1} ou a partida terminará em empate."

**Review by SidiBecker — COMMENTED** [review]
Portuguese translation.

File: translations/get5.phrases.txt (hunk 1 above)
Comment (Saph1s) [inline translations/get5.phrases.txt:74]: "ru" "{1} должны быть готовы в течение {2}, иначе они проиграют матч."

File: translations/get5.phrases.txt (hunk 2 above)
Comment (Saph1s) [inline translations/get5.phrases.txt:79]: "ru" "Команды должны быть готовы в течение {1}, иначе матч закончится вничью."

**Review by Saph1s — COMMENTED** [review]
Russian translation

**Comment by nickdnk** [comment]
Confirmed working, only need translations.

**Pull request merged**